state/applicationoffers_test.go

```diff
@@ -122,13 +121,13 @@ func (s *applicationOffersSuite) TestAddApplicationOffer(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(*offer, jc.DeepEquals, *expectedOffer)
 
-	access, err := s.State.GetOfferAccess(offer.OfferUUID, owner.UserTag())
-	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(access, gc.Equals, permission.AdminAccess)
+	//access, err := s.State.GetOfferAccess(offer.OfferUUID, owner.UserTag())
+	//c.Assert(err, jc.ErrorIsNil)
+	//c.Assert(access, gc.Equals, permission.AdminAccess)
 
-	access, err = s.State.GetOfferAccess(offer.OfferUUID, names.NewUserTag("everyone@external"))
-	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(access, gc.Equals, permission.ReadAccess)
+	//access, err = s.State.GetOfferAccess(offer.OfferUUID, names.NewUserTag("everyone@external"))
+	//c.Assert(err, jc.ErrorIsNil)
+	//c.Assert(access, gc.Equals, permission.ReadAccess)
```

Comment:
Why is this commented out? Remove, or add TODO.

Reply:
Removed, I had left this commented out thinking that it might be useful if people were to try and copy over the tests to the new domain, but in reality we just write new ones.